apps/dapp/src/main.tsx

```diff
@@ -68,6 +69,9 @@ if (env.sentry) {
   });
 }
 
+const { init } = AnalyticsService;
+init();
```

Comment:
Small nit, it doesn't matter here cause it's a static object but doing it this way, instead of just `AnalyticsService.init()` can be problematic. If you "pull off" the method like you did, it loses its scope and has no access to `this`